homeassistant/components/elkm1/__init__.py

```diff
@@ -175,14 +217,16 @@ def __init__(self, element, elk, elk_data):
         """Initialize the base of all Elk devices."""
         self._elk = elk
         self._element = element
+        self._prefix = elk_data['prefix']
         self._temperature_unit = elk_data['config']['temperature_unit']
-        self._unique_id = 'elkm1_{}'.format(
-            self._element.default_name('_').lower())
+        self._unique_id = 'elkm1_{prefix}_{name}'.format(
```

Comment:
Should we remove the extra underscore between prefix and name to make this a non breaking change?